sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/aio/_operations/_operations.py

```diff
@@ -55,14 +57,71 @@
 ClsType = Optional[Callable[[PipelineResponse[HttpRequest, AsyncHttpResponse], T, Dict[str, Any]], Any]]
 
 
-class WebPubSubServiceClientOperationsMixin(MixinABC):
+class WebPubSubServiceClientOperationsMixin(MixinABC):  # pylint: disable=too-many-public-methods
+    @distributed_trace_async
+    async def close_all_connections(  # pylint: disable=inconsistent-return-statements
+        self, *, excluded: Optional[List[str]] = None, reason: Optional[str] = None, **kwargs: Any
+    ) -> None:
+        """Close the connections in the hub.
+
+        Close the connections in the hub.
+
+        :keyword excluded: Exclude these connectionIds when closing the connections in the hub. Default
+         value is None.
+        :paramtype excluded: list[str]
+        :keyword reason: The reason closing the client connection. Default value is None.
+        :paramtype reason: str
+        :return: None
+        :rtype: None
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+        error_map = {
+            401: ClientAuthenticationError,
+            404: ResourceNotFoundError,
+            409: ResourceExistsError,
+            304: ResourceNotModifiedError,
+        }
+        error_map.update(kwargs.pop("error_map", {}) or {})
+
+        _headers = kwargs.pop("headers", {}) or {}
+        _params = kwargs.pop("params", {}) or {}
+
+        cls = kwargs.pop("cls", None)  # type: ClsType[None]
+
+        request = build_close_all_connections_request(
+            hub=self._config.hub,
+            excluded=excluded,
+            reason=reason,
+            api_version=self._config.api_version,
+            headers=_headers,
+            params=_params,
+        )
+        path_format_arguments = {
+            "Endpoint": self._serialize.url("self._config.endpoint", self._config.endpoint, "str", skip_quote=True),
+        }
+        request.url = self._client.format_url(request.url, **path_format_arguments)  # type: ignore
+
+        pipeline_response = await self._client._pipeline.run(  # type: ignore # pylint: disable=protected-access
+            request, stream=False, **kwargs
+        )
+
+        response = pipeline_response.http_response
+
+        if response.status_code not in [204]:
+            map_error(status_code=response.status_code, response=response, error_map=error_map)
+            raise HttpResponseError(response=response)
+
+        if cls:
+            return cls(pipeline_response, None, {})
+
     @distributed_trace_async
     async def get_client_access_token(
         self,
         *,
         user_id: Optional[str] = None,
         roles: Optional[List[str]] = None,
-        minutes_to_expire: Optional[int] = 60,
+        minutes_to_expire: int = 60,
+        group: Optional[List[str]] = None,
```

Comment:
mention the added parameter in the changelog as well

Reply:
here is some pattern to refer: https://github.com/Azure/azure-sdk-for-python/blob/75b9cb73b2c9edfcecaa05c6e03e52a7ea805a30/tools/azure-sdk-tools/packaging_tools/change_log.py#L198-L215

Reply:
Updated